src/Libraries/Analysis/Analysis.csproj

```diff
@@ -62,7 +62,7 @@
   </Target>
   <Target Name="GenerateFiles" AfterTargets="ResolveSateliteResDeps" Condition=" '$(OS)' != 'Unix' ">
     <!-- Generate customization dll -->
-    <GenerateResource SdkToolsPath="$(TargetFrameworkSDKToolsDirectory)" UseSourcePath="true" Sources="$(ProjectDir)AnalysisImages.resx" OutputResources="$(ProjectDir)AnalysisImages.resources" References="$(SystemDrawingDllPath)" />
```

Comment:
so why are these no longer required?

Reply:
There was a [note](https://github.com/DynamoDS/Dynamo/pull/14614/files#diff-bbde35c3169d8580248bcea0e6c8e5722086cba412a65c737fb4c8656d095c55L153) in the props file

The documentation says:
`GenerateResource - References`
```References to load types in .resx files from. .resx file data elements may have a .NET type. When the .resx file is read, this type must be resolved. Typically, it is resolved successfully by using standard type loading rules. If you provide assemblies in References, they take precedence.```
System.Drawing.dll references are resolved automatically from the dotnet shared runtime assemblies (System.Drawing.dll is part of dotnet6/8)
System.Drawing.dll only type forwards types to System.Drawing.Common.dll (which is part of the dynamo build)
So we do not need to add any References to the GenerateResource task